libnd4j/include/ops/ops.h

```diff
@@ -1718,6 +1718,32 @@ namespace simdOps {
         }
     };
 
+    template <typename X>
+    class Mish {
+    public:
+        no_op_exec_special_same
+        no_op_exec_special_same_cuda
+
+        op_def static X op(X d1, X *params) {
+            return d1 * nd4j::math::nd4j_tanh<X,X>(nd4j::math::nd4j_softplus<X,X>(d1));
+        }
+    };
+
+    template <typename X>
+    class MishDerivative {
+    public:
+        no_op_exec_special_same
+        no_op_exec_special_same_cuda
+
+        op_def static X op(X d1, X *params) {
+            auto ex = nd4j::math::nd4j_exp<X,X>(d1);
+            auto e2x = nd4j::math::nd4j_exp<X,X>(2 * d1);
+            auto e3x = nd4j::math::nd4j_exp<X,X>(3 * d1);
```

Comment:
maybe premature optimization, but we could use `e2x = ex * ex` and `e3x = e2x * ex`
Exponent op being expensive compared to multiplication`

Reply:
Done.